Signum.React/Scripts/Lines/AutoCompleteConfig.tsx

```diff
@@ -187,6 +188,18 @@ export class FindOptionsAutocompleteConfig implements AutocompleteConfig<ResultR
         filters = [...defaultFilters, ...filters];
     }
 
+    if (FindOptionsAutocompleteConfig.liteKeyRegEx.test(subStr)) {
+      const lite = parseLite(subStr);
+      if (lite.EntityType.toLowerCase() == qd.queryKey.toLowerCase()) {
+        filters.insertAt(0, {

```

Comment:
This doesn't support enum queries. Use something like getTypeInfos(qd.Colums["Entity"].type) and consider that it could have implementations and then you have to filter by type as well (like `Entity.(Invoice)` is not null)